corehq/apps/es/tests/test_app_adapter.py

```diff
@@ -34,15 +33,6 @@ def test_from_python_works_with_application_dicts(self):
     def test_from_python_raises_for_other_objects(self):
         self.assertRaises(TypeError, app_adapter.from_python, set)
 
-    def test_from_python_is_same_as_transform_app_for_es(self):
-        # this test can be safely removed when transform_app_for_es is removed
-        app_id, app = app_adapter.from_python(self.app)
-        app['_id'] = app_id
-        app_transformed_dict = transform_app_for_es(self.app.to_json())
-        app_transformed_dict.pop('@indexed_on')
-        app.pop('@indexed_on')
-        self.assertEqual(app_transformed_dict, app)
```

Comment:
Do we have equivalent tests (for adapters) to the ones that are being removed in this commit? I think the answer is yes, just confirming.

Reply:
The tests removed are no longer required as they were added to verify the output of `from_python` is the same as `transform_*_to_es` methods.
Now we have removed `transform` functions and `from_python` has been running fine on prod for a while so it makes sense to remove these methods.